protostar/cli/argument_value_from_config_extractor_test.py

```diff
@@ -0,0 +1,81 @@
+from typing import Any, Optional
+
+import pytest
+
+from .argument_value_from_config_extractor import (
+    ArgumentValueFromConfigExtractor,
+    ArgumentValueFromConfigProviderProtocol,
+)
+
+
+class FakeArgumentValueProvider(ArgumentValueFromConfigProviderProtocol):
```

Comment:
I do not like that we have to write so much code to just stub test something. Can we do something about it?

Reply:
I believe it is just not worth to write an unit tests for integration classes. @Arcticae WDYT?

Reply:
it's not that bad if it already exists.